runtime/doc/api-funcs.txt

```diff
@@ -0,0 +1,716 @@
+*api-funcs.txt*         Neovim API Function Reference                   {Nvim}
+
+Note: This documentation is generated from Neovim's API source code.
+
+Contents:
+
+1. Global Functions   |nvim-api-global|
```

Comment:
Let's not prefix the sections with `nvim-`. In general we don't do that anywhere in the help, except in the case of `vim_diff` where we are specifically comparing to Vim.